### ISSUE TITLE
CultureInfo enum

### ISSUE THREAD

**Issue by kelicto** [dotnet/roslyn#58156]
## Background and Motivation

Using enumerations can be done better and more convenient.

`Thread.CurrentThread.CurrentUICultureLanguage = Culture.ZH_CN;`

**Comment by Youssef1313** [comment]
I don't think this is related to dotnet/roslyn. That's a question for dotnet/runtime.

Anyway I don't think it's possible to do that since it's going to be a **big** breaking change.

**Comment by kelicto** [comment]
Please help me to transfer to that project, thanks.
